.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -9,26 +9,26 @@ body:
     attributes:
       label: Python version (`python3 -V`)
       description: Python version (`python3 -V`)
-      placeholder: "3.8"
+      placeholder: "3.10"
```

Comment:
I think we should not have added these contents in the ".github" folder into the repo. These contents are generated by github after the project set up, no need to maintain in the repo ourselves.

Reply:
Discussed with @YuanTingHsieh, we need to have these templates to enable the customized discussions and issues.